### ISSUE TITLE
Multiline string occurs error because of fix_w605

### ISSUE THREAD

**Issue by seeeturtle** [hhatto/autopep8#403]
```python
foo('hello '
    '\world')
```

```python
>>> code = "foo('hello '\n    '\world')\n"
>>> autopep8.fix_code(code, options={'aggressive':1})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bumblebee/envs/autopep8-dev/lib/python3.6/site-packages/autopep8-1.4a1-py3.6.egg/autopep8.py", line 3158, in fix_code
    return fix_lines(sio.readlines(), options=options)
  File "/home/bumblebee/envs/autopep8-dev/lib/python3.6/site-packages/autopep8-1.4a1-py3.6.egg/autopep8.py", line 3233, in fix_lines
    fixed_source = fix.fix()
  File "/home/bumblebee/envs/autopep8-dev/lib/python3.6/site-packages/autopep8-1.4a1-py3.6.egg/autopep8.py", line 572, in fix
    aggressive=self.options.aggressive))
  File "/home/bumblebee/envs/autopep8-dev/lib/python3.6/site-packages/autopep8-1.4a1-py3.6.egg/autopep8.py", line 514, in _fix_source
    modified_lines = fix(result)
  File "/home/bumblebee/envs/autopep8-dev/lib/python3.6/site-packages/autopep8-1.4a1-py3.6.egg/autopep8.py", line 1214, in fix_w605
    tokens = list(generate_tokens(target))
  File "/home/bumblebee/envs/autopep8-dev/lib/python3.6/site-packages/autopep8-1.4a1-py3.6.egg/autopep8.py", line 3993, in generate_tokens
    tokenize.generate_tokens(string_io.readline)
  File "/usr/lib/python3.6/tokenize.py", line 595, in _tokenize
    raise TokenError("EOF in multi-line statement", (lnum, 0))
tokenize.TokenError: ('EOF in multi-line statement', (2, 0))
```
Multiline strings like the above occurs error because of `fix_w605`

python: 3.6.5
autopep8: 1.4a1 (latest version)
pycodestyle: 2.4.0

**Comment by hhatto** [comment]
Thanks for reporting.

It will be fixed this PR(PyCQA/pycodestyle/pull/747) is released.

**Comment by thernstig** [comment]
@hhatto Any idea when pycodestyle will release the new version? :)

**Comment by hhatto** [comment]
pycodestyle has already been released in 2.6.0 and the corresponding autopep8 version 1.5.3.
autopep8 version 1.5.3 doesn't reproduce this problem, so I'll close it.

If the problem seems to reproduce, please reopen the issue.

Thanks
